app/views/content/life-as-a-teacher/explore-subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -1,54 +1,86 @@
     <h2 class="heading--box-blue">Engineering the future</h2>
-    <p>Engineers and physicists help find solutions to some of our biggest problems. From climate change to next generation prosthetics, we need their problem-solving skills and creative thinking to help us survive some of the biggest challenges humanity has ever faced.</p>
+    <p>Engineers and physicists help find solutions to some of our biggest problems. From climate change to next generation prosthetics, we need their problem-solving skills and creative thinking to help us take on some of the biggest challenges humanity has ever faced.</p>
     <p>The more great physics teachers we have in the classroom today, the more children will grow up inspired to become the physicists, engineers and physics teachers of tomorrow.</p>
+    <%= render 'content/shared/teaching/benefits' %>
   </section>
 
   <section class="col col-720">
-    <%= render Content::QuoteComponent.new(
-      text: "My engineering degree provided a natural transition to teaching physics and
-        I value the way science – especially physics – teaches a clear thought process
-        and a way to make rational sense of the world.",
-      name: "Roger, physics teacher",
-      large: false
-    ) %>
+    <%= render 'content/shared/quotes/quote_engineers_teach_physics_roger' %>
   </section>
 
   <section class="col col-720">
     <h2 class="heading--box-blue">Engineers teach physics training programme</h2>
 
     <p>This programme is based on the physics initial teacher training (ITT) course, and has been developed with support from the physics and engineering community. It has been adapted to help you make the transition from engineering or material sciences to teaching physics. The course is designed to capitalise on your experience and empower you with the skills needed to become a great physics teacher.
      </p>
-
-     <p>If you’re concerned about how well you know physics as a subject, do not worry – you’ll be able to do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> that will provide you with the training and support for any gaps in your knowledge.
-     </p>
-
     <p>Your training provider may offer part-time teacher training courses.</p>
 
-     <p>You'll also be invited to an annual community day and have the opportunity to network with existing physics teachers.
+     <p>You'll also be invited to a community day and have the opportunity to network with existing physics teachers.
      </p>
 
     <p><a href="https://find-teacher-training-courses.service.gov.uk/results?engineers_teach_physics=true&study_type%5B%5D=full_time&study_type%5B%5D=part_time&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&degree_required=two_two&applications_open=false&applications_open=true&l=2&subjects%5B%5D=F3">Find engineers teach physics teacher training courses</a>.
```

Comment:
```suggestion
    <p><a href="https://find-teacher-training-courses.service.gov.uk/results?engineers_teach_physics=true&study_type%5B%5D=full_time&study_type%5B%5D=part_time&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&degree_required=two_two&applications_open=false&applications_open=true&l=2&subjects%5B%5D=F3">Find engineers teach physics teacher training courses</a>.
```

If we capitalising Engineers in the course name elsewhere on this page now, shouldn't we here as well?